lib/screens/package_details_page.dart

```diff
@@ -0,0 +1,341 @@
+                    floating: true,
+                    title: Text(
+                      '${args.packageName} '
+                          '${_package.latestVersion.major}'
+                          '.${_package.latestVersion.minor}'
+                          '.${_package.latestVersion.patch}',
+                      style: TextStyle(
+                        color: DynamicTheme.of(context).brightness ==
+                            Brightness.light
+                            ? Colors.black
+                            : Colors.white,
+                      ),
+                    ),
+                    bottom: TabBar(
+                      tabs: _tabs,
+                      controller: _tabController,
+                      isScrollable: true,
+                    ),
+                    actions: <Widget>[
+                      IconButton(
+                        icon: Text(
+                          'API',
+                          style: TextStyle(
+                            color: DynamicTheme.of(context).brightness ==
+                                Brightness.light
+                                ? Colors.black
+                                : Colors.white,
+                          ),
+                        ),
+                        onPressed: () => launch(_package.apiReferenceUrl),
+                      ),
+                      if (_package.repositoryUrl != null) IconButton(
+                        icon: Icon(
+                          Icons.code,
+                          color: DynamicTheme.of(context).brightness ==
+                              Brightness.light
+                              ? Colors.black
+                              : Colors.white,
+                        ),
+                        onPressed: () => launch(_package.repositoryUrl),
+                      ) else Container(),
+                      if (_package.issuesUrl != null) IconButton(
+                        icon: Icon(
+                          Icons.bug_report,
+                          color: DynamicTheme.of(context).brightness ==
+                              Brightness.light
+                              ? Colors.black
+                              : Colors.white,
+                        ),
+                        onPressed: () => launch(_package.issuesUrl),
+                      ) else Container(),
+                      IconButton(
+                        icon: Icon(
+                          Icons.star_border,
+                          color: DynamicTheme.of(context).brightness ==
+                              Brightness.light
+                              ? Colors.black
+                              : Colors.white,
+                        ),
+                        onPressed: () {}, //TODO handle favoriting and unfavoriting packages
+                      ),
+                    ],
+                  ),
+                ];
+              },
+              body: TabBarView(
```

Comment:
Can you separate these into smaller widgets for readability?

Reply:
Sure